src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/Constants.java

```diff
@@ -19,5 +19,8 @@ public class Constants {
 	public static final int INTEGER_ORIGIN_ZERO_VALUE = 0;
 	public static final int LOOP_INC_VALUE = 1;
 	public static final int LOOP_INIT_VALUE = 1;
+	public static final String I18N_LIB_PROPERTIES_PATH = "assets/configuration/i18n/i18n_lib.properties";
+	public static final String LANGUAGE = "French";
```

Comment:
Why is the language hardcoded to `French`?

Reply:
Just there a default language was necessary then I gave a random.

Reply:
BLOCKER: This is not enterprise code quality.

Reply:
And then , how to change language while the FizzBuzzEnterpriseEdition don't need to rerun?

Reply:
I am not an expert on the l18n topic but someone could implement a FizzBuzzLanguageDetectorFactory which will return a FizzBuzzSystemLanguageDetector which then returns the String for the Language.
You would call that in the BuzzStringPrinter and pass the detected Language into the FizzBuzzOutputStrategyl18Proxy. Its really easy

Reply:
That sounds great.The FizzBuzzSystemLanguageDetector will detect the language after running .However ,what if the system language changes ?I'm afraid that the FizzBuzzSystemLanguageDetector detect at all times ,so it would be best that the FizzBuzzSystemLanguageDetector will be notified when the system language changes .

Reply:
Oh thats a real big problem here. If i understand you properly you are unsure how to handle the language change if the FizzBuzz runs over a long time? This is a serious issue. The language should change indeed, so that the calcucation progress of the fizzbuzz software is not lost. The easist way we could archive this is that we exchange the constant language with a proper variable which is not constant. Good point.